[BUG] Errors when compiling / running `build-fzf-tab-module`

# Describe the bug

```
λ zsh -i -c build-fzf-tab-module

(--- 8< ----- 8< ---)

(echo '/* Generated automatically */'; sed -n '/^L/{s/^L//;p;}' < glob.syms) \
		> `echo glob.epro | sed 's/\.epro$/.pro/'`
Updated `zsh.mdh'.
/Applications/Xcode.app/Contents/Developer/usr/bin/make -f Makemod prefix='/usr/local' exec_prefix='/usr/local' bindir='/usr/local/bin' libdir='/usr/local/lib' MODDIR='/usr/local/lib/zsh/5.3.1-dev-0' infodir='/usr/local/share/info' mandir='/usr/local/share/man' datadir='/usr/local/share' fndir='/usr/local/share/zsh/5.3.1-dev-0/functions' htmldir='/usr/local/share/zsh/htmldoc' runhelpdir='/usr/local/share/zsh/5.3.1-dev-0/help' CC='gcc' CPPFLAGS='-I/usr/local/include' DEFS='-DHAVE_CONFIG_H' CFLAGS='-g -Wall -O3' LDFLAGS='-L/usr/local/lib' EXTRA_LDFLAGS='-rdynamic' DLCFLAGS='-fno-common' DLLDFLAGS='-bundle -flat_namespace -undefined suppress' LIBLDFLAGS='' EXELDFLAGS='' LIBS='-liconv -ldl -ltermcap -lm  -lc' DL_EXT='so' DLLD='gcc' AWK='gawk' ANSI2KNR=': ansi2knr' YODL=': yodl ' YODL2TXT=': yodl2txt' YODL2HTML=': yodl2html' FUNCTIONS_INSTALL='' tzsh='zsh' prep
creating Src/aloxaf/Makefile.in
config.status: creating Src/aloxaf/Makefile
echo 'timestamp for *.mdd files' > ../Src/modules.stamp
Updated `fzftab.mdh'.
echo 'timestamp for fzftab.mdh against fzftab.mdd' > fzftab.mdhs
gawk -f ../../Src/makepro.awk fzftab.c Src/aloxaf > fzftab.syms
gawk: ../../Src/makepro.awk:124: warning: regexp escape sequence `\=' is not a known regexp operator
(echo '/* Generated automatically */'; sed -n '/^E/{s/^E//;p;}' < fzftab.syms) \
		> fzftab.epro
(echo '/* Generated automatically */'; sed -n '/^L/{s/^L//;p;}' < fzftab.syms) \
		> `echo fzftab.epro | sed 's/\.epro$/.pro/'`
Updated `fzftab.mdh'.
gcc -c -I. -I../../Src -I../../Src -I../../Src/Zle -I. -I/usr/local/include -DHAVE_CONFIG_H -DMODULE -g -Wall -O3 -fno-common -o fzftab..o fzftab.c
fzftab.c:268:60: warning: passing 'const char *' to parameter of type 'char *' discards qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
        if (name_color[i].pat && pattry(name_color[i].pat, file)) {
                                                           ^~~~
../../Src/pattern.epro:17:60: note: passing argument to parameter 'string' here
extern mod_import_function int pattry _((Patprog prog,char*string));
                                                           ^
1 warning generated.
rm -f fzftab.so
gcc -L/usr/local/lib  -bundle -flat_namespace -undefined suppress -o fzftab.so   fzftab..o    -liconv -ldl -ltermcap -lm  -lc
~
```

## To Reproduce

1. Adding `fzf-tab` to `.zshrc` works as expected:

```
zinit light Aloxaf/fzf-tab
zstyle ':fzf-tab:complete:cd:*' extra-opts --preview=$extract'exa -1 --color=always $realpath'
```

2. Compiling the plugin `zsh -i -c build-fzf-tab-module` (`build-fzf-tab-module` alone yields same result. -- I rarely just run commands to mutate my config, I prefer wrapping it in a dotfile script so changes can be reproduced and script committed to a bare git dotfile repository)

3. (See error)

Deleting the plugin from `.zinit/plugins` and reinstalling yields same result. 
Tried restarting terminal. Tried reboot.

**Moreover, after trying to compile, upon spawning a new shell, `zinit`s next attempt at loading `fzf-tab`:**

```
(anon):3: failed to load module `aloxaf/fzftab': dlopen(/Users/storm/.zinit/plugins/Aloxaf---fzf-tab/modules/Src/aloxaf/fzftab.bundle, 9): image not found
(anon):zmodload:6: no such module aloxaf/fzftab
fzftab module needs to be rebuild, rebuild now?[Y/n]:
```

**Doing that, results in the following:**

```
configuring for zsh 5.3.1-dev-0
checking build system type... x86_64-apple-darwin19.6.0
checking host system type... x86_64-apple-darwin19.6.0
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.ou
... -------- 8< -------- 8< -----------
```

**Note the first line!:** `configuring for zsh 5.3.1-dev-0` !

Relevant?

```
▐λ▍/usr/local/bin/zsh --version
zsh 5.8 (x86_64-apple-darwin19.3.0)
▐λ▍/bin/zsh --version
zsh 5.7.1 (x86_64-apple-darwin19.0)
▐λ▍zsh --version
zsh 5.8 (x86_64-apple-darwin19.3.0)
▐λ▍which -a zsh
/usr/local/bin/zsh
/bin/zsh
```

## Expected behavior

Expected compile to succeed and new binary implemented automatically.

## Environment

 - OS: `10.15.6 build 19G73` same result on `iMacPro1,1` and `MacBookPro10,1`
 - `zsh 5.8 (x86_64-apple-darwin19.3.0)`
 - `iTerm2 Build 3.3.12`
 - fzf-tab version  _:latest_ from repo (via `zinit`)

## `.zshrc`

*(Part of the shell initialization, like envvars and ensuring `tag` exists, done in `.zshenv` before this `.zshrc` is sourced:)*

```
# ~/.zshrc (haakonstorm/dotfiles)

typeset -U path=( $path $(cat $zconf/zenv/PATH ) ) 
export PATH

function include-zshrc() {
  for FILE in $( /usr/local/bin/tag --match .zshrc ${1}/* ); do source $FILE; done; }

include-zshrc ${zconf}/zenv
include-zshrc ${zconf}/zfunc
include-zshrc ${zconf}/zali

source "${ZSH}/oh-my-zsh.sh"
source "${HOME}/.iterm2_shell_integration.zsh"

__jump_chpwd() {
  jump chdir
}
jump_completion() {
  reply=(${(f)"$(jump hint $@)"})
}
j() {
  local dir="$(jump cd $@)"
  test -d "$dir" && export THEDIR="$dir" && cd "$THEDIR" && h1 $(subdir "${THEDIR}") && echo "——————————[ $dir ]———————————"
}

typeset -gaU chpwd_functions
chpwd_functions+=__jump_chpwd
compctl -U -K jump_completion j
setopt inc_append_history
setopt share_history

eval "$(_HASS_CLI_COMPLETE=source_zsh hass-cli)"
eval "$(pyenv init -)" 
eval "$(rbenv init -)"
eval "$(hub alias -s)"
eval "$(direnv hook zsh)"

source <(kubectl completion zsh)
complete -F __start_kubectl k

source "$HOME/.zinit/bin/zinit.zsh"
autoload -Uz _zinit
(( ${+_comps} )) && _comps[zinit]=_zinit

unalias run-help
autoload run-help

source ~/.config/gitstatus/gitstatus.prompt.zsh

function my_set_prompt() {
  RPROMPT=''
  if gitstatus_query MY && [[ $VCS_STATUS_RESULT == ok-sync ]]; then
    RPROMPT=${${VCS_STATUS_LOCAL_BRANCH:-@${VCS_STATUS_COMMIT}}//\%/%%}  # escape %
    (( $VCS_STATUS_NUM_STAGED    )) && RPROMPT+='+'
    (( $VCS_STATUS_NUM_UNSTAGED  )) && RPROMPT+='!'
    (( $VCS_STATUS_NUM_UNTRACKED )) && RPROMPT+='?'
  fi
  setopt noprompt{bang,subst} promptpercent  
}
gitstatus_stop 'MY' && gitstatus_start -s -1 -u -1 -c -1 -d -1 'MY'
autoload -Uz add-zsh-hook
add-zsh-hook precmd my_set_prompt
prompt_update_last_dir () {
    echo "${PWD}">${PWDTF}
    echo "${PWD}">${PWDF}
    open -j -g "bitbar://refreshPlugin?name=3env.sh"
}
precmd_functions+=(prompt_update_last_dir)
setopt promptsubst
function retcode() {}
PROMPT=$'%{\e[1;33m%}▐%{\e[0m%}%{\e[1;30m%}%{\e[43m%}λ%{\e[0m%}%{\e[1;33m%}▍%{\e[0m%}'

_tmutil(){
  list=$( tmutil | /usr/bin/grep -v ^$ | /usr/bin/grep ^Usage | awk '{print $3}' )
  COMPREPLY=( $(compgen -W "$list" ${COMP_WORDS[COMP_CWORD]}  ) )
}
complete -F _tmutil tmutil

zinit light Aloxaf/fzf-tab
zstyle ':fzf-tab:complete:cd:*' extra-opts --preview=$extract'exa -1 --color=always $realpath'

zinit light zsh-users/zsh-autosuggestions

zinit ice wait silent blockf atpull'zinit creinstall -q .'
zinit light zsh-users/zsh-completions

zinit ice wait silent atinit"zpcompinit; zpcdreplay"
zinit light zdharma/fast-syntax-highlighting 

zinit light skywind3000/z.lua
function _z() { _zlua "$@"; }
FZ_HISTORY_CD_CMD=_zlua

zinit ice atload"zpcdreplay" atclone'./zplug.zsh'
zinit light g-plane/zsh-yarn-autocompletions

zinit light changyuheng/fz

zinit light denisidoro/navi

setopt NO_HUP

CDPATH=:/🐚 

source /usr/local/opt/resty/share/resty/resty
source /usr/local/opt/modules/init/zsh

my-accept-line () {  
  if [[ "${BUFFER:0:1}" == '|' ]]; then    
    /usr/local/bin/cliclick kd:shift,cmd t:a ku:shift w:100 t:c ku:cmd
    pbpaste>"${THATF}"
    BUFFER='cat ${THATF} '${BUFFER} "string"
  fi
  zle .accept-line
}
zle -N accept-line my-accept-line

fpath=( "${zfunc}" $fpath )
autoload -Uz fzf-kubectl-config-use-context

[ -f ~/.fzf.zsh ] && source ~/.fzf.zsh 

RG_PREFIX='rg --column --line-number --no-heading --color=always --smart-case '
INITIAL_QUERY=''
FZF_DEFAULT_COMMAND="$RG_PREFIX '$INITIAL_QUERY' $HOME"
__fif() { fzf --bind "change:reload:$RG_PREFIX {q} $HOME || true" --ansi --phony --query "$INITIAL_QUERY" | cut -d ':' -f1 }
find-in-files() { LBUFFER="${LBUFFER}$(__fif)"; local ret=$?; zle reset-prompt; return $ret }
zle -N find-in-files
bindkey '^f' find-in-files
```

--- a/fzf-tab.zsh
+++ b/fzf-tab.zsh
@@ -595,8 +595,12 @@ toggle-fzf-tab() {
 }
 
 build-fzf-tab-module() {
+  local MACOS
+  if [[ ${OSTYPE} == darwin* ]]; then
+    MACOS=true
+  fi
   pushd $FZF_TAB_HOME/modules
-  CPPFLAGS=-I/usr/local/include CFLAGS="-g -Wall -O3" LDFLAGS=-L/usr/local/lib ./configure --disable-gdbm --without-tcsetpgrp
+  CPPFLAGS=-I/usr/local/include CFLAGS="-g -Wall -O3" LDFLAGS=-L/usr/local/lib ./configure --disable-gdbm --without-tcsetpgrp ${MACOS:+DL_EXT=bundle}
   make -j
   popd
 }

